app/controllers/stats_controller.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class StatsController < ApplicationController
+  before_action :authenticate_user!
+
+  def total_watch_time
+    total_watch_time = VideoWatchEvent.where(user: current_user).sum(:duration).round
+
+    render json: { "stats": { "time_watched": total_watch_time } }
```

Comment:
Quotes keys are something specific for JSON. This value is a Ruby Hash, so use symbols as keys:

```suggestion
    render json: { stats: { time_watched: total_watch_time } }
```